src/Platforms/Echo.Platforms.AsmResolver/CilInstructionFormatter.cs

```diff
@@ -0,0 +1,14 @@
+﻿using AsmResolver.PE.DotNet.Cil;
+using Echo.ControlFlow.Serialization.Dot;
+
+namespace Echo.Platforms.AsmResolver
+{
+    /// <summary>
+    /// Provides a custom formatter for <see cref="CilInstruction"/>s.
+    /// </summary>
+    public class CilInstructionFormatter : IInstructionFormatter<CilInstruction>
+    {
+        /// <inheritdoc />
+        public string Format(in CilInstruction instruction) => $"{instruction.OpCode.Mnemonic} {instruction.Operand}";
```

Comment:
AsmResolver defines a [ICilInstructionFormatter](https://github.com/Washi1337/AsmResolver/blob/master/src/AsmResolver.PE/DotNet/Cil/ICilInstructionFormatter.cs) interface, with a default implementation [CilInstructionFormatter](https://github.com/Washi1337/AsmResolver/blob/master/src/AsmResolver.DotNet/Code/Cil/CilInstructionFormatter.cs), which has the advantage that it works better with formatting the operand than just calling `ToString()` on `CilInstruction.Operand`.

Perhaps we can use this implementation of `IInstructionFormatter` of Echo as an adapter to `ICilInstructionFormatter` from AsmResolver.